Editor/nadena.dev.modular-avatar.core.editor.asmdef

```diff
@@ -43,6 +43,11 @@
             "name": "com.vrchat.avatars",
             "expression": "",
             "define": "MA_VRCSDK3_AVATARS"
+        },
+        {
+            "name": "com.vrchat.avatars",
+            "expression": "3.5.2",
+            "define": "MA_VRCSDK3_AVATARS_3_5_2_OR_NEWER"
```

Comment:
OK for now but I'll probably make this a minimum requirement before 1.10.0 release.